tests/manage/noobaa/test_create_bucket.py

```diff
@@ -53,7 +55,7 @@ class TestBucketCreation:
         condition=config.ENV_DATA['platform'] != 'AWS',
         reason="Tests are not running on AWS deployed cluster"
     )
-    def test_s3_bucket_creation(self, noobaa_obj, buckets):
+    def s3_bucket_creation(self, noobaa_obj, buckets):
```

Comment:
wouldn't be better to use some pytest mark to skip this tests (or remove the tier1 mark), instead of renaming it? (So it will be still clear, that it is test and not some auxiliary function.)

Reply:
done